### ISSUE TITLE
Question about the paper

### ISSUE THREAD

**Issue by jihoontack** [universome/inr-gan#7]
Thank you for the awesome work, and I really enjoyed reading it!

I have some questions while reading your paper.

In Section 4.2, you have explored the properties of the implicit neural representation (INR).

Is this property hold for any INR (assuming that the INR shares the same architecture) or INR generated by your framework?
Especially, the meaningful interpolation part (Figure 5) was truly a surprising observation, and does it hold for naively trained INR?

Best,
Jihoon

**Comment by universome** [comment]
Hi! We didn't try, but I am pretty sure that this property will **not** hold for a random pair of INRs trained with the reconstruction loss to represent an image. I believe that there should either be some parameter sharing, or fine-tuning (like in https://github.com/justinpinkney/toonify) or smth else.

**Comment by jihoontack** [comment]
Also agree with you. Thank you for the reply!
